docs/developers/linea-version/index.mdx

```diff
@@ -13,6 +13,34 @@ import TokenBridgeL2 from "!!raw-loader!/files/testnet/2023-06-13-release/TokenB
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
+## Beta v1
+
+**Mainnet: TBC**
+
+**Linea Sepolia: September 23**
+
+### Arithmetization
```

Comment:
I'd suggest to abstract the content overall; saying something like: 
Linea's Beta release aims at proving 100% of the EVM specification

This release is a first intermediary step, it includes: 
- New arithmetization, with selected modules activated from the Linea spec (insert link to spec repo)
- Transition to Besu for trace generation